docs/resources/lts_keywords_alarm_rule.md

```diff
@@ -0,0 +1,178 @@
+  The [KeywordsRequests](#KeywordsAlarmRule_KeywordsRequests) structure is documented below.
+
+* `frequency` - (Required, List) Specifies the alarm frequency configurations.
+  The [Frequency](#KeywordsAlarmRule_Frequency) structure is documented below.
+
+* `alarm_level` - (Required, String) Specifies the alarm level.
+  The value can be: **INFO**, **MINOR**, **MAJOR** and **CRIRICAL**.
+
+* `description` - (Optional, String) Specifies the description of the keywords alarm rule.
+
+* `send_notifications` - (Optional, Bool, ForceNew) Specifies whether to send notifications.
+  Changing this parameter will create a new resource.
+
+* `notification_rule` - (Optional, List, ForceNew) Specifies the notification rule.
+  The [NotificationRule](#KeywordsAlarmRule_NotificationRule) structure is documented below.
+  Changing this parameter will create a new resource.
+
+* `trigger_condition_count` - (Optional, Int) Specifies the count to trigger the alarm.
+  Defaults to **1**.
+
+* `trigger_condition_frequency` - (Optional, Int) Specifies the frequency to trigger the alarm.
+  Defaults to **1**.
+
+* `send_recovery_notifications` - (Optional, Bool) Specifies whether to send recovery notifications.
+
+* `recovery_frequency` - (Optional, Int) Specifies the frequency to recover the alarm.
+  Defaults to **3**.
+
+* `status` - (Optional, String) Specifies the status. The value can be: **RUNNING** and **STOPPING**.
+  Defaults to **RUNNING**.
+
+<a name="KeywordsAlarmRule_KeywordsRequests"></a>
+The `KeywordsRequests` block supports:
+
+* `keywords` - (Required, String) Specifies the keywords.
+
+* `condition` - (Required, String) Specifies the keywords request condition.
+  The value can be: **>=**, **<=**, **<** and **>**.
+
+* `number` - (Required, Int) Specifies the line number.
+
+* `log_stream_id` - (Required, String) Specifies the log stream id.
+
+* `log_group_id` - (Required, String) Specifies the log group id.
+
+* `search_time_range_unit` - (Required, String) Specifies the unit of search time range.
+  The value can be: **minute** and **hour**.
+
+* `search_time_range` - (Required, Int) Specifies the search time range.
+  When the `search_time_range_unit` is **minute**, the value ranges from **1** to **60**.
+  When the `search_time_range_unit` is **hour**, the value ranges from **1** to **24**
+
+<a name="KeywordsAlarmRule_Frequency"></a>
+The `Frequency` block supports:
+
+* `type` - (Required, String) Specifies the frequency type.
+  The value can be: **CRON**, **HOURLY**, **DAILY**, **WEEKLY** and **FIXED_RATE**.
+
+* `cron_expression` - (Optional, String) Specifies the cron expression.
+  This parameter is used when `type` is set to **CRON**.
+
+* `hour_of_day` - (Optional, Int) Specifies the hour of day.
+  This parameter is used when `type` is set to **DAILY** or **WEEKLY**.
+  The value ranges from **0** to **23**.
+
+* `day_of_week` - (Optional, Int) Specifies the day of week.
+  This parameter is used when `type` is set to **WEEKLY**.
```

Comment:
The value ranges from **1** to **7**.

please make sure 1 is Sunday or Monday

Reply:
sure